base_contact/i18n/base_contact.pot

```diff
@@ -0,0 +1,191 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* base_contact
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2014-09-19 22:10+0000\n"
+"PO-Revision-Date: 2014-09-19 18:10-0500\n"
+"Last-Translator: EL Hadji DEM <elhadji.dem@savoirfairelinux.com>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Poedit 1.5.4\n"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "City"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "other position"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contacts"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid ""
+"To see personal information about this contact, please go to to the his "
+"person form:"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Street..."
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "State"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "at"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Tags..."
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Other Positions"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Phone:"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Company"
+msgstr ""
+
+#. module: base_contact
+#: field:res.partner,contact_id:0
+msgid "Main Contact"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Fax:"
+msgstr ""
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:31
+#, python-format
+msgid "Standalone Contact"
+msgstr ""
+
+#. module: base_contact
+#: help:res.partner,nationality_id:0
+msgid "Nationality."
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Address"
+msgstr ""
+
+#. module: base_contact
+#: field:res.partner,nationality_id:0
+msgid "Nationality"
+msgstr ""
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:32
+#, python-format
+msgid "Attached to existing Contact"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "ZIP"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Country"
+msgstr ""
+
+#. module: base_contact
+#: field:res.partner,birthdate_date:0
+msgid "Birthdate"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Person"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contact"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Mobile:"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr ""
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```

Comment:
Better remove this line that can be conflictive (it's a bug in the translation export)